examples_tests/npm-non-monorepo.t

```diff
@@ -0,0 +1,5 @@
+  $ . ${TESTDIR}/setup.sh non-monorepo npm
+# run twice and make sure it works
+  $ npx turbo build lint > /dev/null
```

Comment:
`turbo` is a dev dependency for the `non-monorepo` example, is there a reason you're using `npx` here?

Looks like that was existing behavior though.

Reply:
yeah, i'm not sure why it's like that. I don't see an explanation or conversation about it where it was added https://github.com/vercel/turbo/pull/2650. cc @tknickman if you remember!